test/ad_service/math/basic_test.exs

```diff
@@ -5,20 +5,23 @@ defmodule AdService.Math.BasicTest do
     ad_details = [
       %AdService.Advertisement{
         total_spend: Decimal.new(100),
+        ecpm: Decimal.new(2.50),
         body: "ad body",
         campaign_id: "9839afe6-5ac3-4443-be3c-dbb7a2af01e6",
         headline: "ad headline",
         image_url: "http://example.com"
       },
       %AdService.Advertisement{
         total_spend: Decimal.new(50),
+        ecpm: Decimal.new(3.00),
         body: "ad body",
         campaign_id: "9839afe6-5ac3-4443-be3c-dbb7a2af01e7",
         headline: "ad headline",
         image_url: "http://example.com"
       },
       %AdService.Advertisement{
         total_spend: Decimal.new(25),
```

Comment:
If we aren't going to reference total_spend from this struct we should drop it from the object.